proto/meta.proto

```diff
@@ -547,3 +547,34 @@ message GetServingVnodeMappingsResponse {
 service ServingService {
   rpc GetServingVnodeMappings(GetServingVnodeMappingsRequest) returns (GetServingVnodeMappingsResponse);
 }
+
+message EventLog {
+  enum EventType {
+    UNSPECIFIED = 0;
+    META_NODE_START = 1;
+    CREATE_STREAM_JOB_FAIL = 2;
+  }
+  // Event logs identifier, which should be populated by event log service.
+  optional string unique_id = 1;
+  // Processing time, which should be populated by event log service.
+  optional uint64 timestamp = 2;
+  EventType event_type = 3;
+  string info = 4;
+}
+
+message AddEventLogRequest {
+  repeated EventLog event_logs = 1;
+}
+
+message AddEventLogResponse {}
+
```

Comment:
As mentioned in https://github.com/risingwavelabs/risingwave/pull/13392#issuecomment-1809858692, I'd like to keep the event log limited as much as possible in order to avoid potential abusing. For the RPC interface, I think we may remove them for now, unless we see any necessity.

Reply:
> unless we see any necessity.

It can simplify implementation. It would be more convenient to allow sending event log from compute node at any time on demand, rather than piggyback errors through existing RPCs' response and handle them case by case in meta node.

For example, to report failure causing recovery, at least error from several RPCs like BuildActors, InjectBarrier and BarrierComplete of StreamService need be handled. By contrast, compute node actually does have recorded all unexpected actor failure locally [here](https://github.com/risingwavelabs/risingwave/blob/9ca614a8031e5a63624ce1109b71fb22b51a8004/src/stream/src/task/barrier_manager/managed_state.rs#L70), so compute node can send one event log per recovery based on it, if allowed.